server/channels/store/storetest/property_value_store.go

```diff
@@ -899,6 +942,97 @@ func testSearchPropertyValues(t *testing.T, _ request.CTX, ss store.Store) {
 	}
 }
 
+func testSearchPropertyValuesSince(t *testing.T, _ request.CTX, ss store.Store) {
+	// Create values with controlled timestamps for precise testing
+	baseTime := model.GetMillis()
+	groupID := model.NewId()
+
+	// Create values with specific timestamps
+	valueEarly := &model.PropertyValue{
+		GroupID:    groupID,
+		TargetID:   model.NewId(),
+		TargetType: "test_type",
+		FieldID:    model.NewId(),
+		Value:      json.RawMessage(`"early value"`),
+	}
+	_, err := ss.PropertyValue().Create(valueEarly)
+	require.NoError(t, err)
+	// Set specific timestamp
+	valueEarly.UpdateAt = baseTime
+
+	time.Sleep(10 * time.Millisecond) // Ensure different timestamps
+
+	valueMid := &model.PropertyValue{
+		GroupID:    groupID,
+		TargetID:   model.NewId(),
+		TargetType: "test_type",
+		FieldID:    model.NewId(),
+		Value:      json.RawMessage(`"mid value"`),
+	}
+	_, err = ss.PropertyValue().Create(valueMid)
+	require.NoError(t, err)
+	// Set specific timestamp
+	valueMid.UpdateAt = baseTime + 1000
+
+	time.Sleep(10 * time.Millisecond)
+
+	valueLate := &model.PropertyValue{
+		GroupID:    groupID,
+		TargetID:   model.NewId(),
+		TargetType: "test_type",
+		FieldID:    model.NewId(),
+		Value:      json.RawMessage(`"late value"`),
+	}
+	_, err = ss.PropertyValue().Create(valueLate)
+	require.NoError(t, err)
+	// Set specific timestamp
+	valueLate.UpdateAt = baseTime + 2000
+
+	// Update the values with controlled timestamps
+	_, err = ss.PropertyValue().Update("", []*model.PropertyValue{valueEarly, valueMid, valueLate})
```

Comment:
Beware that the `Update` method overwrites the `UpdateAt` for the properties that saves, and the test is assuming that they're set based on the `baseTime`

Reply:
wow, good catch! I have re-made the test to make it have more sense.